version

```diff
@@ -1 +1 @@
-0.22.0
+0.23.0
```

Comment:
Bumping this is done by our CI system. Any reason why it was done here?

Reply:
^^ @jpluscplusm @richardTowers @keymon - poke.

Reply:
I think that is wrong, I will drop it.